samples/uri/web/GreatNotes/StoreApp/ActionsManifest.json

```diff
@@ -0,0 +1,92 @@
+{
+  "version": 2,
+  "actions": [
+    {
+      "id": "AProtocol.OpenImage",

```

Comment:
Do we need `usesGenerativeAI` now that we're in version two? I think all of these actions will be dropped as is